source/swap/test/Swap.js

```diff
@@ -1,14 +1,14 @@
 const Swap = artifacts.require('Swap')
 const Types = artifacts.require('Types')
 const FungibleToken = artifacts.require('FungibleToken')
-const NonFungibleToken = artifacts.require('NonFungibleToken')
-const OMGToken = artifacts.require('OMGToken')
-
+const TransferHandlerRegistry = artifacts.require('TransferHandlerRegistry')
+const ERC20TransferHandler = artifacts.require('ERC20TransferHandler')
```

Comment:
Testing with OMG also feels like a good idea. If we remove all cases except standard ERC20 tokens from our tests, how do we know that the contracts handle anything else?

Reply:
These tests have been moved to transfers/TransferHandler.js